src/Plugins/RpcServer/Model/BlockHashOrIndex.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2015-2024 The Neo Project.
+//
+// BlockHashOrIndex.cs file belongs to the neo project and is free
+// software distributed under the MIT software license, see the
+// accompanying file LICENSE in the main directory of the
+// repository or http://www.opensource.org/licenses/mit-license.php
+// for more details.
+//
+// Redistribution and use in source and binary forms with or without
+// modifications are permitted.
+
+namespace Neo.Plugins.RpcServer.Model;
+
+public class BlockHashOrIndex
```

Comment:
Why you not use type of `T` and just pass the value of the class.

Reply:
Why can't i do it this way?

Reply:
BTW, we dont really know the ype when we use it......, it might be int or string, thus can not general type here.